index.html

```diff
@@ -565,6 +565,7 @@
 The time immediately after the user agent receives the last byte of the response or immediately before the transport connection is closed, whichever comes first. The resource here can be received either from <a

```

Comment:
This was introduced following https://github.com/w3c/resource-timing/issues/25 , ie to sync with nav-timing. And this was in nav-timing since October 2010 ( https://dvcs.w3.org/hg/webperf/rev/3bb1ac67bfb2 ), based on the f2f ( https://lists.w3.org/Archives/Public/public-web-perf/2010Oct/0016.html ). It says "Add support for scenario where connection is closed."